docs/socketexample.rst

```diff
@@ -552,7 +569,7 @@ need this method.
 .. doctest::
 
    >>> utils = list(zope.component.getAllUtilitiesRegisteredFor(IUSSocket))
-   >>> utils.sort()
+   >>> utils.sort(key=lambda x: type(x).__name__)
```

Comment:
The reason I didn't do that here was that the use of the key function means that `sorted` would either be one really long line or get broken into two, which looks bad (and is annoying hard to indent):
```pycon
>>> utils = list(zope.component.getAllUtilitiesRegisterFor(IUSSocket),
...               key=lambda x: type(x).__name__)
```